### DISCUSSION TITLE
Yarn Workspaces protocol In peerDependencies

### DISCUSSION THREAD

**Discussion by alizeait** [yarnpkg/berry#2734] [Q&A] [answered]
Setting the workspace protocol(`workspace:*`) in `peerDependencies` seems to be reverted by yarn upon install to just `*`, is this an expected behavior or a bug that needs attention?

**Comment by arcanis** [comment, marked as answer]
That's expected. Peer dependencies only support semver versions, since they describe what you expect to be provided, not the location from which it should be obtained.

**Reply by alizeait** [reply]
Makes sense, thanks @arcanis
